docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -45,6 +45,8 @@ The following metadata keys are always required for the survey runner, they do n
   The url of the account service (i.e. rrm or ras) used to launch the survey
 ``response_id``
   A unique identifier for the questionnaire response
+``party_id``
+  The ID of the party allocated to the responding unit.
```

Comment:
Businesses are more commonly referred to as reporting units.  I'd change this to.
```
The ID of the party allocated to the reporting unit.
```

Reply:
What's the expectation and use cases for `party_id` in EQ v3. I'm not sure we have anything on the backlog around this?

Reply:
It's needed to display a link on EQ to the new help section in RASRM. I guess it'll be needed for EQ v3 too, might be worth catching up with @gareth-poole-ons

Reply:
@pricem14pc we're not going to do this anymore, instead we'll expose a URL that EQ v2 can use to route to the new help section. That URL will take ru ref and survey id as parameters which they already in the payload.